Gruntfile.js

```diff
@@ -378,6 +378,22 @@ module.exports = function( grunt ) {
 							}
 						},
 						{
+							name: "globalize.unit",
```

Comment:
GitHub has some kind of issue with this file. Looking at the source locally, I also get messed up syntax highlighting in my editor (SublimeText). Looks like an issue with the regex on line 192, completely unrelated to this PR though. Somehow this helps:
```diff
diff --git a/Gruntfile.js b/Gruntfile.js
index ce458fc..60819b1 100644
--- a/Gruntfile.js
+++ b/Gruntfile.js
@@ -189,7 +189,7 @@ module.exports = function( grunt ) {
                             // Remove browserify wrappers.
                             .replace( /^\(function\(f\){if\(typeof exports==="object"&&type.*/, "" )
                             .replace( "},{}],2:[function(require,module,exports){", "" )
-                            .replace( /},{"\.\/messageformat-parser":1,"make-plural\/plural.*/, "" )
+                            .replace( /},{"\.\/messageformat-parser\":1,"make-plural\/plural.*/, "" )
                             .replace( /},{}\]},{},\[2\]\)\(2\)[\s\S]*?$/, "" )
 
                             // Set `MessageFormat.plurals` and remove `make-plural/plurals`
```